backend/internal/processes/bootstrap.go

```diff
@@ -36,94 +36,109 @@ func BootstrapEnv(appconf *config.AppConfig) {
 		if err != nil {
 			log.WithError(err).Error("Problem writing to auditlog")
 		}
-		return
+		return err
 	}
 
+	log.Infof("Provisioning S3 Bucket")
 	err = provisionS3(appconf)
 	if err != nil {
 		log.WithError(err).Error("Error provisioning S3 bucket")
 		err = store.AddToAudit(application.Bootstrap_Unsuccessful, "test")
 		if err != nil {
 			log.WithError(err).Error("Problem writing to auditlog")
 		}
-		return
+		return err
 	}
+
+	log.Infof("Setting Up Default SSM Parameters")
 	err = storeDefaultSSMParameters(appconf, store)
 	if err != nil {
 		log.WithError(err).Error("Error setting SSM Parameters")
 		err = store.AddToAudit(application.Bootstrap_Unsuccessful, "test")
 		if err != nil {
 			log.WithError(err).Error("Problem writing to auditlog")
 		}
-		return
+		return err
 	}
+
+	log.Infof("Setting Up Terraform")
 	err = initTerraform(store, appconf)
 	if err != nil {
 		log.WithError(err).Error("Error installing Terraform")
 		err = store.AddToAudit(application.Bootstrap_Unsuccessful, "test")
 		if err != nil {
 			log.WithError(err).Error("Problem writing to auditlog")
 		}
-		return
+		return err
 	}
 
 	//r := action.ActRunnerImpl{}
 	//err = UpdateCoreConfig(appconf, store, nil, "")
 	//if err != nil {
 	//	log.WithError(err).Error("Problem updating ssm config")
 	//}
+
+	log.Infof("Setting Up HTTPD Gateway from Marketplace")
 	err = installGateway(store, appconf)
 	if err != nil {
 		log.WithError(err).Error("Error installing HTTPD Gateway")
 		err = store.AddToAudit(application.Bootstrap_Unsuccessful, "test")
 		if err != nil {
 			log.WithError(err).Error("Problem writing to auditlog")
 		}
-		return
+		return err
 	}
 
+	log.Infof("Setting Up Health Status Lambda")
 	err = installHealthStatusLambda(store, appconf)
 	if err != nil {
 		log.WithError(err).Error("Error installing Health Status")
 		err = store.AddToAudit(application.Bootstrap_Unsuccessful, "test")
 		if err != nil {
 			log.WithError(err).Error("Problem writing to auditlog")
 		}
-		return
+		return err
 	}
 
+	log.Infof("Setting Up Basic API Gateway from Marketplace")
 	err = installBasicAPIGateway(store, appconf)
 	if err != nil {
 		log.WithError(err).Error("Error installing API Gateway")
 		err = store.AddToAudit(application.Bootstrap_Unsuccessful, "test")
 		if err != nil {
 			log.WithError(err).Error("Problem writing to auditlog")
 		}
-		return
+		return err
 	}
 
+	log.Infof("Setting Up Unity UI from Marketplace")
 	err = installUnityUi(store, appconf)
 	if err != nil {
 		log.WithError(err).Error("Error installing unity-ui")
 		err = store.AddToAudit(application.Bootstrap_Unsuccessful, "test")
 		if err != nil {
 			log.WithError(err).Error("Problem writing to auditlog")
 		}
-		return
+		return err
 	}
 
 	err = store.AddToAudit(application.Bootstrap_Successful, "test")
 	if err != nil {
 		log.WithError(err).Error("Problem writing to auditlog")
+		return err
 	}
 
+	log.Infof("Bootstrap Process Completed Succesfully")
+	return nil
```

Comment:
Is this return statement necessary?  There wasn't a return here previously, and I just wasn't sure if this was a best practice, or it may change some behavior?

Reply:
oh never mind, I see why you are returning now